.github/workflows/go.yml

```diff
@@ -30,5 +30,8 @@ jobs:
     - name: Build
       run: make build
 
+    - name: Build Multi-arch
```

Comment:
Do we need both this and the prior action? Since this will cover the linux/amd64 target (the default) it should be fine to have just this one. So I'd say replace the previous action with this one.